types/relay-runtime/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
```

Comment:
Also add `"strictFunctionTypes": true,`

Reply:
@voxmatt This will require this branch to first be rebased (after merging https://github.com/voxmatt/DefinitelyTyped/pull/2 please 😄) and then add this to the tsconfig files.